src/lsst/cst/conversions/tools.py

```diff
@@ -51,214 +46,87 @@ def data_id_to_str(data_id: dict):
-    df = df.sort_values(sort_key, ascending=ascending)
-    if set_index:
-        df.set_index(np.array(range(len(df))), inplace=True)
-    return df
+    data_id_str = json.dumps(data_id)
+    return data_id_str
 
 
-def shuffle_dataframe(df: pd.DataFrame, random_state: int = 0) -> pd.DataFrame:
-    """Return a copy of the dataframe df shuffled, random_state
-    argument may be used to reproduce same shuffling.
+def psf_size_at_pixel_xy(psf, bbox, xy):
+    """Obtains the size of the PSF in an image
+    at a given xy coordinate.
 
     Parameters
     ----------
-    df: `pandas.DataFrame`
-        dataframe to be shuffled
-
-    random_state: `int`
-        number to reproduce same randomness
+    psf : `lsst.meas.extensions.psfex.PsfexPsf` `lsst.meas.algorithms.CoaddPsf`
+        PSF object for a calexp or deepCoadd; from .getPsf().
+    bbox : `lsst.geom.Box2I`
+        Bounding box for the calexp or deepCoadd; from .getBBox().
+    xy : `Tuple[int, int]`
+        Pixel coordinates x and y where PSF size is to be evaluated.
 
     Returns
     -------
-    result: `pandas.DataFrame`
-        shuffled dataframe
+    psf_size: `dict`
+        Size of the PSF in pixels; sigma and FWHM.
     """
-    df_randomized = df.sample(frac=1, random_state=random_state)
-    return df_randomized
+    point2I = geom.Point2I(xy[0], xy[1])
+    if bbox.contains(point2I):
+        point2D = geom.Point2D(xy[0], xy[1])
+        sigma = psf.computeShape(point2D).getDeterminantRadius()
+        fwhm = sigma * 2.0 * np.sqrt(2.0 * np.log(2.0))
+    else:
+        raise Exception("Coordinates xy not contained by image boundaries.")
+    psf_size = {'sigma': sigma, 'fwhm': fwhm}
+    return psf_size
 
 
-class TractPatchInformation:
-    """Tract patch information
-
-    Parameters
-    ----------
-    patch: `int`
-        Patch value.
-    tract: `int`
-        Tract value.
-    """
-
-    def __init__(self, tract: int, patch: int):
-        self._tract = tract
-        self._patch = patch
-
-    @property
-    def patch(self):
-        """Patch value.
-
-        Returns
-        -------
-        patch: `int`
-            Patch value.
-        """
-        return self._patch
-
-    @property
-    def tract(self):
-        """Patch value.
-
-        Returns
-        -------
-        tract: `int`
-            Tract value.
-        """
-        return self._tract
-
-    def to_dict(self):
-        """Dictionary containing
-        Tract-patch information
-
-        Returns
-        -------
-        values: `dict[str, int]`
-            Dictionary with tract-patch information.
-        """
-        return dict(tract=self._tract, patch=self._patch)
-
-    def __str__(self):
-        return f"tract: {self._tract} patch: {self._patch}"
-
-    def __repr__(self):
-        return str(self)
-
-
-def tract_patch_from_ra_dec(ra: float, dec: float):
-    """Look for nearest tract-patch information from a
-    coordinate ra-dec.
+def nearest_patch_from_ra_dec(ra, dec):
```

Comment:
Add argument type and method return:

def nearest_patch_from_ra_dec(ra: x, dec: y) -> X:

Reply:
Updated to be `ra: float, dec: float`